sdks/java/nexmark/pom.xml

```diff
@@ -25,6 +25,10 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
+  <properties>
+    <calcite.version>1.13.0</calcite.version>
```

Comment:
<!--new_thread; commit:964cc527e15c17e26b9584aaaf993cddc225398b; resolved:0-->
This variable is not referenced here. The version in sdks/java/extendsion/sql/pom.xml is already this. So is it already solved by a prior PR? Or did I just not follow the flow of this variable?